packages/explorer/src/EmbeddedExplorer.ts

```diff
@@ -29,6 +30,8 @@ export interface BaseEmbeddableExplorerOptions {
   persistExplorerState?: boolean; // defaults to 'false'
 
   endpointUrl: string;
+  subscriptionUrl?: string;
+  protocol?: GraphQLSubscriptionLibrary;
```

Comment:
didn't we decide that we don't need these as inputs at all since we aren't supporting `handleSubscriptionRequest` from the config right now? I think we can remove these and just use the ones sent by `studio-ui`s post messages. This means manual schema entry won't be able to use subscriptions, which I think is a good trade off for this complexity, since we never supported it anyways. 

So if we know that a user has manually sent us a schema, we can return back an error when we get a PM asking us to perform a subscription